core/domain/improvements_services_test.py

```diff
@@ -453,6 +453,7 @@ def test_put_for_updated_task_entries_without_changing_last_updated(self):
 
         model = improvements_models.TaskEntryModel.get_by_id(task_entry.task_id)
         self.assertEqual(model.resolver_id, None)
+        self.assertEqual(model.resolver_id, None)
```

Comment:
Isn't this just a duplicated check? Please remove.

Reply:
Fixed.